gems/sorbet-runtime/test/types/props/decorator.rb

```diff
@@ -172,18 +172,6 @@ class InnerStruct < T::Struct
     prop :the_hash, T::Hash[String, InnerStruct]
   end
 
-  describe 'validating prop values' do
```

Comment:
Instead of deleting all these tests, can we convert them into tests that capture the new behavior?